### PR TITLE
Update torchscript README

### PR DESCRIPTION
**Pull request by xcjason**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

1. Please confirm that if this PR changes any image versions, then that's the sole change this PR makes.

**Release note**:

```release-note

```

### CODE REVIEW

File: docs/samples/v1beta1/triton/torchscript/README.md [hunk 1]
```diff
@@ -333,24 +333,24 @@ The transformer does not enforce a specific schema like predictor but the genera
 {
   "instances": [
     {
-      "image": { "b64": "aW1hZ2UgYnl0ZXM=" },
+      "image_bytes": { "b64": "aW1hZ2UgYnl0ZXM=" },
       "caption": "seaside"
     },
     {
-      "image": { "b64": "YXdlc29tZSBpbWFnZSBieXRlcw==" },
```
Comment (xcjason) [inline docs/samples/v1beta1/triton/torchscript/README.md:340]: This keyword is needed at https://github.com/kserve/kserve/blob/master/docs/samples/v1beta1/triton/torchscript/image_transformer_v2/image_transformer_v2.py#L31

Comment (xcjason) [reply docs/samples/v1beta1/triton/torchscript/README.md:340]: These `b64` bytes actually crashes https://github.com/kserve/kserve/blob/master/docs/samples/v1beta1/triton/torchscript/image_transformer_v2/image_transformer_v2.py#L32, throwing error `OSError: cannot identify image file`. A random guess is Pillow updated its code and try to validate if bytes are image files. Tested with `b64` bytes from real images could work. But the strings are much longer.

**Comment by yuzisun** [comment]
Thanks @xcjason !

/lgtm
/approve

**Pull request merged**